Unsplash/Unsplash/Scene/Home/HomeViewController.swift

```diff
@@ -2,27 +2,220 @@
 //  HomeViewController.swift
 //  Unsplash
 //
-//  Created by Nha Pham on 8/25/20.
+//  Created by Nha Pham on 8/17/20.
 //  Copyright © 2020 Nha Pham. All rights reserved.
 //
 
 import UIKit
-
+import Network
 protocol HomeViewControllerDelegate: class {
 }
 
 final class HomeViewController: UIViewController {
+    var isfetching = false
+    var isLoadedData = false
+    var isHomeAppear = false
+    var networkManager = NetworkManager.shared
+    var viewModel = HomeViewModel()
+    var headerView: HeaderCollectionReusableView?
     weak var delegate: HomeViewControllerDelegate?
+    @IBOutlet weak var homeCollectionView: UICollectionView!
```

Comment:
```suggestion
    @IBOutlet private weak var homeCollectionView: UICollectionView!
```
để các Outlet lên đầu